src/graphql/helpers.ts

```diff
@@ -431,7 +431,8 @@ async function getControllerDomains(address: string): Promise<string[]> {
       },
       body: JSON.stringify({
         method: 'lookup_domains',
-        params: address
+        params: address,
+        network: process.env.NETWORK === 'mainnet' ? '1' : '11155111'
```

Comment:
It is actually `livenet` https://hub.snapshot.org/api/, better to look for `testnet`
```suggestion
        network: network === 'testnet' ? '11155111' : '1'
```